src/main/scala/org/broadinstitute/dsde/workbench/leonardo/auth/ServiceAccountProviderHelper.scala

```diff
@@ -31,7 +35,11 @@ object ServiceAccountProviderHelper {
   }
 }
 
-class ServiceAccountProviderHelper(wrappedServiceAccountProvider: ServiceAccountProvider, config: Config) extends ServiceAccountProvider(config) with LazyLogging {
+class ServiceAccountProviderHelper(wrappedServiceAccountProvider: ServiceAccountProvider, config: Config)(implicit system: ActorSystem)
+  extends ServiceAccountProvider(config) with FutureSupport with LazyLogging {
+
+  private lazy val providerTimeout = config.getAs[FiniteDuration]("providerTimeout").getOrElse(30 seconds)
```

Comment:
drive-by nitpick: if we put these default values in `reference.conf` we shouldn't need `getOrElse` everywhere

Reply:
True, this is double-defaulted. Will remove the default here.